flink-python/pyflink/datastream/connectors/cassandra.py

```diff
@@ -0,0 +1,1648 @@
+        ...    .add_contact_points("127.0.0.1") \\
+        ...    .with_port(9876) \\
+        ...    .with_cluster_name("cluster_name") \\
+        ...    .with_credentials("user", "password")  \\
+        ...    .with_ssl()
+        """
+        JSimpleClusterBuilder = get_gateway().jvm.org.apache.flink.streaming.connectors.\
+            cassandra.SimpleClusterBuilder
+        return ClusterBuilder(JSimpleClusterBuilder())
+
+    def with_cluster_name(self, name: str) -> 'ClusterBuilder':
+        """
+        An optional name for the creation cluster.
+
+        Note: this is not related to the Cassandra cluster name (though you are free to provide the
+        same name).
+        """
+        self._j_cluster_builder.withClusterName(name)
+        return self
+
+    def with_port(self, port: int) -> 'ClusterBuilder':
+        """
+        The port to use to connect to the Cassandra host.
+
+        If not set through this method, the default port (9042) will be used instead.
+        """
+        self._j_cluster_builder.withPort(port)
+        return self
+
+    def allow_beta_protocol_version(self) -> 'ClusterBuilder':
+        """
+        Create cluster connection using latest development protocol version, which is currently in
+        beta. Calling this method will result into setting USE_BETA flag in all outgoing messages,
+        which allows server to negotiate the supported protocol version even if it is currently in
+        beta.
+
+        This feature is only available starting with version V5.
+
+        Use with caution, refer to the server and protocol documentation for the details on latest
+        protocol version.
+        """
+        self._j_cluster_builder.allowBetaProtocolVersion()
+        return self
+
+    def with_max_schema_agreement_wait_seconds(self, max_schema_agreement_wait_seconds: int) \
+            -> 'ClusterBuilder':
+        """
+        Sets the maximum time to wait for schema agreement before returning from a DDL query.
+
+        If not set through this method, the default value (10 seconds) will be used.
+        """
+        self._j_cluster_builder.withMaxSchemaAgreementWaitSeconds(max_schema_agreement_wait_seconds)
+        return self
+
+    def with_protocol_version(self, version: ProtocolVersion) -> 'ClusterBuilder':
+        """
+        The native protocol version to use.
+        """
+        self._j_cluster_builder.withProtocolVersion(version._to_j_protocol_version())
+        return self
+
+    def add_contact_point(self, address: Union[str, EndPoint]) -> 'ClusterBuilder':
+        """
+        Adds a contact point by str or EndPoint.
+        """
+        if isinstance(address, str):
+            self._j_cluster_builder.addContactPoint(address)
+        else:
+            self._j_cluster_builder.addContactPoint(address._j_end_point)
+        return self
+
+    def add_contact_points(self, *addresses: Union[str, InetAddress]) -> 'ClusterBuilder':
```

Comment:
ditto

Reply:
This is the part about ClusterBuilder, I will create a new PR and fix it for easy review~